### PR TITLE
Add test to build the default site

### PR DESCRIPTION
**Pull request by benbalter**
Motivated by https://github.com/jekyll/jekyll/issues/5153, and inspired by our tests on the Pages Gem, this PR adds a purposely (for now) failing test to generate and build Jekyll's default site.

The test is quick to run (2.3s, including Ruby's boot on my machine), and would provide a good smoke test / sanity check that the default Jekyll action works out of the box.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ env:
   matrix:
     - TEST_SUITE=test
     - TEST_SUITE=cucumber
-
+    - TEST_SUITE=default-site
```
Comment (parkr) [inline .travis.yml]: Does this need to run 3 times, once per Ruby version?

Comment (benbalter) [inline .travis.yml]: Your call. I'm fine with it running once? If so, what's the best way to do that?

Comment (parkr) [inline .travis.yml]: To add it only once, choose your fave Ruby version and add it to the `matrix.include` section above this one (see `fmt`):

```yaml
- rvm: 2.3.0
  env: TEST_SUITE=test-jekyll-new
```

Or whatever 😄

**Comment by parkr** [comment]
Travis is doing weird things. :cool:

@jekyllbot: merge +dev

**Pull request merged**